README.rst

```diff
@@ -183,9 +183,7 @@ Add the following settings to ``settings.py``:
     from django.conf.urls import include, url
     from django.contrib.staticfiles.urls import staticfiles_urlpatterns
 
-    from openwisp_utils.admin_theme.admin import admin, openwisp_admin
-
-    openwisp_admin()
+    from openwisp_utils.admin_theme.admin import admin
```

Comment:
the standard way to import admin is `from django.contrib`, eg: https://github.com/openwisp/openwisp-controller/blob/master/tests/openwisp2/urls.py#L4

Can you change it like that please?

Reply:
Yes, that's what I had first (see https://github.com/openwisp/openwisp-controller/commit/00159f7d0ff4fb2f141da7eeeba70aca62279e25), but I changed it because `ansible-openwisp2` (https://github.com/openwisp/ansible-openwisp2/blob/master/templates/openwisp2/urls.py) imports from `openwisp_utils.admin_theme.admin`.

Currently it's the same because `openwisp_utils.admin_theme.admin` also imports from `django.contrib`. A problem would be if that would change.

Reply:
I must have been lazy here: https://github.com/openwisp/ansible-openwisp2/commit/46c2fa202a41e58af9208ae25d84e9df3243da3e
But let's change it. It's better to import the django admin via django. We may need to change again the `openwisp_utils` internals and if we don't have such imports lying around we will be able to change it more easily.

Reply:
I changed it in ansible-openwisp2: https://github.com/openwisp/ansible-openwisp2/commit/e6f45dc6db033f67377278c9bb882263fe28d601

Let me know if you spot it anywhere else.